service/surf/settings/base.py

```diff
@@ -86,7 +86,6 @@
 ]
 
 SESSION_COOKIE_SECURE = PROTOCOL == "https"
-CSRF_COOKIE_SECURE = PROTOCOL == "https"
```

Comment:
We need to keep this. Not for the API, but for the admin

Reply:
✅